.latexmkrc

```diff
@@ -6,6 +6,11 @@ push @generated_exts, 'glo', 'gls', 'glg';
 push @generated_exts, 'acn', 'acr', 'alg';
 push @generated_exts, 'ntn', 'not', 'nlg';
 
+$aux_dir = 'aux';
+$out_dir = 'aux';
+$pdflatex = 'xelatex --shell-escape --interaction=nonstopmode %O %S';
```

Comment:
@eyalroz I noticed you removed somewhere the `--interaction=nonstopmode` flag that I suggested adding here. Was this intentional? I find it really annoying to interact with XeLaTeX when there's an error, and I can't imagine any user out there would desire the default behavior. What do you think?

Reply:
Yes, that was intentional. Sorry, I should have mentioned it. I don't see a significant benefit in proceeding past the first error - enough to make it the default. With `--interaction=nonstopmode`, the user will the need to search through the log to find the errors, particularly the first one. I definitely want stop-on-error to be the default behavior.

Reply:
> With `--interaction=nonstopmode`, the user will the need to search through the log to find the errors, particularly the first one. I definitely want stop-on-error to be the default behavior.

I totally agree and understand, please see the addition of `--halt-on-error` in #70 which should implement exactly what you are aiming at.